src/main/java/net/mangolise/gamesdk/instance/InstanceAnalysis.java

```diff
@@ -104,7 +104,7 @@ private static Int2ObjectMap<List<Point>> computeChunk(int minSection, PolarChun
 
             if (palette.length == 1) {
                 String stateString = palette[0];
-                int stateId = namespace2stateId.computeIntIfAbsent(stateString, ignored -> safeGetStateId(stateString));
```

Comment:
This should just be a freestanding commit in the future, I'm not a fan of mixing in the unrelated changes, but it's fine for now.

Reply:
I figured they were deprecated by the new Minestom version and so felt relevant